library/webpack-loaders/css-loader.js

```diff
@@ -5,6 +5,12 @@ const genericNames = require('generic-names')
 const { findCssGlobalFiles } = require('../lib/findCssGlobalFiles')
 const vm = require('vm')
 
+const {
+  kaliber: {
+    nativeCssCustomProperties = false
+  } = {}
+} = require('@kaliber/config')
+
```

Comment:
Dit moet via de `loaderOptions` worden meegegeven.

Reply:
Maar hoe maken we dat dan configurabel? Of gaan we native custom properties forceren als onderdeel van een updaten naar de volgende @kaliber/build versie? 

Misschien mis ik iets, zover ik kan zien geef dan bij `moduleOptions` aan wanneer dat van toepassing is, maar dat gebeurd daar voornamelijk op filename.

Reply:
@peeke In `lib/build.js` wordt alle configuratie uitgelezen en de loaders geconfigureerd. Daar wordt bijvoorbeeld ook `minifyOnly` gezet.